ts/screens/main/MessagesScreen.tsx

```diff
@@ -67,26 +75,58 @@ class MessagesScreen extends React.Component<Props, never> {
     ) : null;
   };
 
-  private renderMockedMessages = (
+  public renderItem = (messageDetails: IMessageDetails) => {
+    return (
+      <MessageComponent
+        key={messageDetails.item.id}
+        date={messageDetails.item.created_at}
+        services={this.props.services}
+        senderServiceId={messageDetails.item.sender_service_id}
```

Comment:
the lookup of the service from the ID shouldn't be a responsibility of this component - `MessageComponent` should not even be aware of where the services data come from - @fpersico